internal/service/domain/auth/user_test.go

```diff
@@ -291,3 +291,61 @@ func TestLogoutRequest(t *testing.T) {
 		assert.Error(t, err)
 	})
 }
+
+func TestGetAllUsers(t *testing.T) {
+	t.Run("should return all users successfully", func(t *testing.T) {
+		// given
```

Comment:
Maybe add a test case »should return empty slice when user are found«? Same goes for the handler.

Reply:
I think, when **no** users are found.